src/script.js

```diff
@@ -1,3 +1,50 @@
 "use strict";
 
 // TODO - write your code here.
+const randomDamage = () => {
+    return Math.floor(Math.random() * 10) +1;
+}
+console.log(randomDamage)
+
+const chooseOption = (opt1, opt2) => {
+    const randNum = Math.round(Math.random()); 
+    return randNum === 0 ? opt1 : opt2;
+}
+
+const attackPlayer = (health) => {
+    return health - randomDamage()
+}
+
+const logHealth = (player, health) => {
+    console.log(`${player} health: ${health}`)
+}
+
+const logDeath = (winner, loser) => {
```

Comment:
I appreciate the empty line between each function. I think that makes your code nice and readable.